### PR TITLE
Fix CAPS demo

### PR DESCRIPTION
**Pull request by DarthMax**
- PatternGraph creation works correctly
- Fix pretty printing plans

### CODE REVIEW

File: caps-cora/src/main/scala/org/opencypher/caps/impl/flat/FlatPlanner.scala [hunk 1]
```diff
@@ -35,7 +35,9 @@ class FlatPlanner extends DirectCompilationStage[LogicalOperator, FlatOperator,
         producer.cartesianProduct(process(lhs), process(rhs))
 
       case logical.Select(fields, graphs, in, _) =>
-        val withAliasesRemoved = producer.removeAliases(fields, process(in))
+        val withAliasesRemoved = if(fields.nonEmpty) {
```
Comment (s1ck) [inline caps-cora/src/main/scala/org/opencypher/caps/impl/flat/FlatPlanner.scala]: style: space after if

File: caps-cora/src/main/scala/org/opencypher/caps/impl/record/RecordHeader.scala [hunk 2]
```diff
@@ -148,9 +148,10 @@ final case class RecordHeader(internalHeader: InternalHeader) extends CypherReco
   }
 
   override def toString: String = {
-    val s = slots
-    s"RecordHeader with ${s.size} slots: \n\t ${slots.mkString("\n\t")}"
+    s"RecordHeader with ${slots.size} slots"
```
Comment (s1ck) [inline caps-cora/src/main/scala/org/opencypher/caps/impl/record/RecordHeader.scala]: one line

File: caps-ir/src/main/scala/org/opencypher/caps/ir/api/IRElement.scala [hunk 3]
```diff
@@ -44,12 +44,16 @@ trait IRGraph extends IRElement {
   def toNamedGraph: IRNamedGraph = IRNamedGraph(name, schema)
 
   def schema: Schema
+
+  override def toString: String = s"IRNamedGraph(name= $name)"
```
Comment (s1ck) [inline caps-ir/src/main/scala/org/opencypher/caps/ir/api/IRElement.scala]: space before =

File: caps-ir/src/main/scala/org/opencypher/caps/ir/api/IRElement.scala [hunk 4]
```diff
@@ -44,12 +44,16 @@ trait IRGraph extends IRElement {
...
 }
 
 final case class IRNamedGraph(name: String, schema: Schema) extends IRGraph {
   override def toNamedGraph: IRNamedGraph = this
 }
 
-final case class IRExternalGraph(name: String, schema: Schema, uri: URI) extends IRGraph
+final case class IRExternalGraph(name: String, schema: Schema, uri: URI) extends IRGraph {
+  override def toString: String = s"IRExternalGraph(name= $name, uri= $uri)"
```
Comment (s1ck) [inline caps-ir/src/main/scala/org/opencypher/caps/ir/api/IRElement.scala]: space before =

File: caps-spark/src/test/scala/org/opencypher/caps/demo/GCDemoTest.scala [hunk 5]
```diff
@@ -36,6 +36,7 @@ class GCDemoTest extends CAPSTestSuite with SparkSessionFixture with Neo4jServer
   protected override def hdfsURI: URI = new URIBuilder(super.hdfsURI).setScheme("hdfs+csv").build()
   protected override val dfsTestGraphPath = "/csv/prod"
 
+
```
Comment (s1ck) [inline caps-spark/src/test/scala/org/opencypher/caps/demo/GCDemoTest.scala]: remove blank line

File: caps-spark/src/test/scala/org/opencypher/caps/impl/spark/CAPSPatternGraphTest.scala [hunk 6]
```diff
@@ -512,7 +512,7 @@ class CAPSPatternGraphTest extends CAPSTestSuite with GraphCreationFixture {
         |RETURN GRAPH result OF (a)-[k]->(b)
       """.stripMargin)
 
-    when.graphs("result").asInstanceOf[CAPSPatternGraph].baseTable.data.count() should equal(1)
+    when.graphs("result").asInstanceOf[CAPSPatternGraph].baseTable.data.count() should equal(3)
```
Comment (Mats-SX) [inline caps-spark/src/test/scala/org/opencypher/caps/impl/spark/CAPSPatternGraphTest.scala]: So... this is switching over to bag semantics for `GRAPH OF`? The same triple of `a, k, b` is present three times in the pattern graph it seems? Note that there is only a single `:KNOWS` relationship in the base graph.

**Pull request merged**